File: Exercises/Foundational/Basic-Level/2/ts/calculator.ts
import { createInterface } from "readline/promises";
async function calc(): Promise<void> {
  const input = createInterface({
    input: process.stdin,
    output: process.stdout,
  });
  const first_num = await input.question("Enter first number: ");
  const second_num = await input.question("Enter second number: ");
  const sum = +first_num + +second_num;
  const subtraction = +first_num - +second_num;
  const multiplication = +first_num * +second_num;
  const division = +first_num / +second_num;
  console.log(`${first_num} + ${second_num} = ${sum}`);
  console.log(`${first_num} - ${second_num} = ${subtraction}`);
  console.log(`${first_num} * ${second_num} = ${multiplication}`);
  console.log(`${first_num} / ${second_num} = ${division}`);
  input.close();
}
(async () => {
  await calc();
})();

File: Exercises/Foundational/Basic-Level/2/fortran/calculator.f90
PROGRAM Calculator
    IMPLICIT NONE
    REAL :: first_num, second_num
    REAL :: sum, sub, mul, div

    PRINT *, "Enter the first number: "
    READ *, first_num

    PRINT *, "Enter the second number: "
    READ *, second_num

    sum = first_num + second_num
    sub = first_num - second_num
    mul = first_num * second_num
    div = first_num / second_num
    PRINT *, first_num, " + ", second_num, " = ", sum
    PRINT *, first_num, " - ", second_num, " = ", sub
    PRINT *, first_num, " * ", second_num, " = ", mul
    PRINT *, first_num, " / ", second_num, " = ", div

END PROGRAM Calculator
